bin/preview.sh

```diff
@@ -22,6 +22,20 @@ if [[ -n "$CENTER" && ! "$CENTER" =~ ^[0-9] ]]; then
 fi
 CENTER=${CENTER/[^0-9]*/}
 
+#MS Win support
+if [[ $FILE =~ "\\" ]]; then
```

Comment:
* Put a space after `#` for consistency
* Let's use single quotes to avoid escaping.

```sh
# MS Windows support
if [[ $FILE =~ '\' ]]; then
```